pandas/tools/tests/test_util.py

```diff
@@ -89,6 +90,14 @@ def test_set_locale(self):
         self.assertEqual(current_locale, CURRENT_LOCALE)
 
 
+class TestToNumeric(tm.TestCase):
+    def test_basic_conversion(self):
+        raw = pd.Series(['1','2.0',3])
```

Comment:
move to the testing above (e.g. ``test_util``) with the other tests for ``to_numeric``